### DISCUSSION TITLE
no stdout output when run on group of machine

### DISCUSSION THREAD

**Discussion by NatchanonBR** [CrowdStrike/psfalcon#418] [Q&A]
I manage POS machines and need to check certain services on the POS machines. However, there are around 4xxx machines. 
When I run the command below, there is no stdout output shown in the Excel file; it appears as blank spaces.

CheckServices is a PowerShell script that, if it finds the required service, it will display information indicating that the service was found. If not, it will show that the service was not found.

I’m not sure if there are any techniques that could help with this ?

Example command
Invoke-FalconRtr runscript -CloudFile="CheckServices" -HostIds $HostIds -Timeout '600' | Export-csv d:\Save-Result.csv

**Comment by bk-cs** [comment]
I see a syntax error in your example; `-CloudFile="CheckServices"` is not properly quoted. You also don't need to quote `Timeout`. It should look like this:
```
Invoke-FalconRtr runscript '-CloudFile="CheckServices"' -HostId $HostIds -Timeout 600 | Export-csv d:\Save-Result.csv
```
Assuming that didn't resolve it...

* Do you receive other output in the CSV?
* What does the output look like if you test on a small number of machines and don't output to CSV (i.e. is there anything under `$rtr.stdout` if you run this command)?
```
$rtr = Invoke-FalconRtr runscript '-CloudFile="CheckServices"' -HostId $HostIds -Timeout 600
```

**Comment by NatchanonBR** [comment]
Thanks for the suggestion. 
But it seems like I have to run a small number of machines, about 800 at a time, to get the stdout output data to be displayed in the file.

**Reply by bk-cs** [reply]
Are you getting no results at all? Try lowering your timeout.
